src/stream/stream/partial_cmp.rs

```diff
@@ -0,0 +1,93 @@
+use std::cmp::Ordering;
+use std::pin::Pin;
+
+use super::fuse::Fuse;
+use crate::prelude::*;
+use crate::future::Future;
+use crate::stream::Stream;
+use crate::task::{Context, Poll};
+
+// Lexicographically compares the elements of this `Stream` with those
+// of another.
+#[derive(Debug)]
```

Comment:
I think that we hide docs for all `Future` types

```suggestion
#[doc(hidden)]
#[allow(missing_debug_implementations)]
```